### DISCUSSION TITLE
WordPress Plugin Directory

### DISCUSSION THREAD

**Discussion by stavros-prh** [gdarko/wp-batch-processing#14] [General]
Hi,
First, love the plugin as it quickly got me through a sticky problem.
Second, is there any intention to publish this plugin in the official WordPress Plugin Directory?  I'm sure many others would benefit from it being there.
-Stavros

**Comment by gdarko** [comment]
Hi Stavros,

I will check if such plugins are possible to be published and if it is i will do.

Best,
Darko
